mesonbuild/interpreter/interpreter.py

```diff
@@ -1276,7 +1276,7 @@ def func_add_languages(self, node: mparser.FunctionNode, args: T.Tuple[T.List[st
 
         if disabled:
             for lang in sorted(langs, key=compilers.sort_clink):
-                mlog.log('Compiler for language', mlog.bold(lang), 'skipped: feature', mlog.bold(feature), 'disabled')
+                mlog.debug('Compiler for language', mlog.bold(lang), 'skipped: feature', mlog.bold(feature), 'disabled')
```

Comment:
As a feature check that most likely represents parts of the build getting disabled, I think maybe this should stay an mlog.log, what do you think?